### PR TITLE
Mount a persistent block device

### PR DESCRIPTION
**Pull request by mgax**
Say we want to persist some data from a vagrant run to another. For example, a copy of a large production database. A matching PR is waiting in `vagrant-vmck` (https://github.com/vmck/vagrant-vmck/pull/18) but it's blocked on https://github.com/vmck/vagrant-vmck/pull/17.

### CODE REVIEW

File: vmck/backends/qemu.py [hunk 1]
```diff
@@ -73,6 +73,17 @@ def task_group(job, options):
         '-device', 'virtio-9p-pci,fsdev=vmck,mount_tag=vmck',
     ]
 
+    if options['usbstick']:
+        usbsticks = Path(settings.QEMU_USBSTICKS).resolve()
+        image = (usbsticks / options['usbstick']).resolve()
+        assert usbsticks in image.parents
```
Comment (gabriel-v) [inline vmck/backends/qemu.py]: This seems like a no-op if the `.resolve()` calls above don't throw. How about `image = (Path.QEMU_USBSTICKS / options['usbstick']).resolve(); assert image.exists()`?

File: vmck/backends/qemu.py [hunk 2]
```diff
@@ -73,6 +73,17 @@ def task_group(job, options):
         '-device', 'virtio-9p-pci,fsdev=vmck,mount_tag=vmck',
     ]
 
+    if options['usbstick']:
```
Comment (gabriel-v) [inline vmck/backends/qemu.py]: Since the QEMU option name is `usb-storage`, could we use "storage" instead of "stick" for the name?

**Comment by mgax** [comment]
So https://passthroughpo.st/disk-passthrough-explained/ says the USB driver is slow and we should use virtio instead. Please don't merge yet.

File: vmck/backends/qemu.py (hunk 1 above)
Comment (mgax) [reply vmck/backends/qemu.py]: Its only purpose is to make sure the user doesn't escape outside the images folder. I'll add a note to the assert.

File: vmck/backends/qemu.py (hunk 2 above)
Comment (mgax) [reply vmck/backends/qemu.py]: Done.

**Comment by mgax** [comment]
Closing, since nobody still wants this feature.

**Pull request closed without merging**